### PR TITLE
docs: update installation command

### PR DESCRIPTION
**Pull request by snitin315**
Refers https://github.com/eslint/eslint/issues/16050

We decided to use `npm init @eslint/config`.

Not sure if `@` is clear in this font.

Screenshot 2022-07-04 at 4 30 55 PM

Latest screenshot - 

Screenshot 2022-07-18 at 6 29 53 PM

### CODE REVIEW

**Comment by nzakas** [comment]
Hmmm yeah. I wonder if we can just Courier for the “@“ symbol?

**Comment by snitin315** [comment]
I tried with [@font-face](https://css-tricks.com/modifying-specific-letters-with-css-and-javascript/#aa-font-face) but wasn't able to make it work 😕 

This is how it looks with `font-family: var(--display-font)`:

Screenshot 2022-07-06 at 9 11 01 AM

**Review by nzakas — COMMENTED** [review]
This looks nice but I’m worried we are sacrificing accessibility with this approach. I didn’t realize we were using an input field, but it makes sense because keyboard users can tab into the field to read and select text. We may need to think a bit more about this change.

File: src/assets/scss/homepage.scss [hunk 1]
```diff
@@ -46,11 +46,21 @@
...
     }
 
-    input {
+    .code-snippet {
         all: unset;
         min-width: 30ch;
         color: inherit;
-        font-family: var(--mono-font);      
+        font-family: var(--mono-font);
+        display: inline-block;
```
Comment (amareshsm) [inline src/assets/scss/homepage.scss:54]:
```suggestion
```

**Pull request closed without merging**